tutor/plugins/v0.py

```diff
@@ -282,7 +282,6 @@ class OfficialPlugin(BasePlugin):
     NAMES = [
         "android",
         "discovery",
-        "ecommerce",
```

Comment:
not sure if this should be removed from v0 as it can serve some backwards compatibility purposes.

Reply:
yes, this needs to be discussed...

Reply:
This class is only meant for the discovery of plugins in the official github-built binary (see `make bundle`). We don't want the ecommerce plugin there, at least not in the main branch, so we should remove it from this list.